post.html

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+---
+
+<article class="post">
+{{ content }}
+
+<footer class="card">
+    <ol class="chips   flex-row flex-wrap flex-horiz-center flex-vert-center">
+        {% for tag in page.categories %}
+        <li><span class="chip" data-tag-name="{{ tag }}">{{ tag }}</span></li>
+        {% endfor %}
+    </ol>
+
+    <ol class="plain dates-and-times">
+        <li>
+            <time class="date" datetime="{{ page.date | date_to_xmlschema }}">
```

Comment:
perhaps now's a good time to make this an object rather than a value

Reply:
See also: https://github.com/BlueHuskyStudios/Neon-Jekyll-Basics/pull/1/files#diff-7c7f3a79fdc0c409d35806a8b59fe522R43

Reply:
Nope turns out Jekyll reserves `date`. :/